src/frontend/src/functions/applicationperiodinfo.js

```diff
@@ -0,0 +1,149 @@
+import {
+    checkValidity,
+    getApplicationPeriodInfoJson,
+    getOngoingApplicationPeriodInfoJson,
+    getPersonalApplicationPeriodInfoJson,
+    getToken,
+    setApplicationPeriodInfoJson,
+    setOngoingApplicationPeriodInfoJson,
+    setPersonalApplicationPeriodInfoJson
+} from "./localstorage";
+import axios from "axios";
+import {url} from "./url";
+import {notification} from "antd";
+
+
+export async function submitApplicationPeriod(application_window_open, application_window_close,
+                                              applicable_periods, applicable_rooms, applicable_students) {
+    var data = JSON.stringify({
+        "application_window_open": application_window_open,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/MarkHershey/SUTDHousingPortal/pullRequest?prid=7298734)
```suggestion
        application_window_open,
```